workflow_scripts/check_model.py

```diff
@@ -17,7 +17,12 @@ def run_backend_ort(model_path, test_data_set=None):
         return
     # if 'test_data_set_N' doesn't exist, create test_dir
     if not test_data_set:
-        onnxruntime.InferenceSession(model_path)
+        # Start from ORT 1.10, ORT requires explicitly setting the providers parameter if you want to use execution providers
+        # other than the default CPU provider (as opposed to the previous behavior of providers getting set/registered by default
+        # based on the build flags) when instantiating InferenceSession.
+        # Following code assumes NVIDIA GPU is available, you can specify other execution providers or don't include providers parameter
+        # to use default CPU provider.
+        onnxruntime.InferenceSession(model_path, providers=['CUDAExecutionProvider'])
```

Comment:
If ONNX model zoo decides to verify models with both CPU and GPU provider, then we should keep `onnxruntime.InferenceSession(model_path)` and `onnxruntime.InferenceSession(model_path, providers=['CUDAExecutionProvider'])` with corresponding packages.